README.md

```diff
@@ -18,106 +20,89 @@ The core of the module is a NodeJS Lambda function. This function is intended to
 
 ## Installation
 
-WebProducer can be installed as a runtime dependency using your favourite package manager.
+WebProducer can be installed as a runtime dependency by adding to your package.json
```

Comment:
Suggested change
```suggestion
 WebProducer Rocks
```

Reply:
@webdivelement I'm not going to hit the "commit" button for obvious reasons but this is kinda cool. In the comment, I added a codefence with the suggestion keyword
 ```suggestion
new code I want to suggest goes here```

And it is now offered as a single click to implement to the PR author.